codec_specs.md

```diff
@@ -14,6 +14,46 @@ Important Note:
 
 The intention behind this list is `NOT` to list all existing audio and video codecs, but rather to list those codecs that are `currently supported` in Matroska and therefore need a well defined codec ID so that all developers supporting Matroska will use the same ID. If you feel we missed support for a very important codec, please tell us on our development mailing list (cellar at ietf.org).
 
+## Defining Matroska Codec Support
+
+Support for a codec is defined in Matroska with the following values.
+
+### Codec ID
+
+Each codec supported for storage in Matroska MUST have a unique Codec ID. Each Codec ID MUST be prefixed with a two character string according to the associated type of the codec as listed in the following table.
+
+Codec Type | Codec ID Prefix
+-----------|----------------
+Video      | V_
+Audio      | A_
+Subtitle   | S_
+Button     | B_
+
+Each Codec ID MUST then use a `Major Codec ID` and may optionally use a `Minor Codec ID` and `Micro Codec ID` with a forward-slash (`/`) delimiter between each component.
```

Comment:
Ugh. We've never deprecated the IDs officially, nor have we officially added the ID that's actually used — `A_AAC`. I clearly remember there being a discussion and consensus either on the mailing list or on the IRC channel about this change.

I don't know of a single Matroska muxer that still writes those old IDs.

I'd consider them de facto deprecated, with the specs page needing to catch up.

I like both of the descriptors you've proposed.